src/main/js/main_html5.js

```diff
@@ -935,11 +932,43 @@ require("../../../html5-common/js/utils/environment.js");
       if (OO.isSafari && _video && _video.textTracks) {
         _video.textTracks.onchange = onTextTracksChange;
       }
+
+      if (_video.audioTracks) {
+        _video.audioTracks.onchange = _onAudioChange;
+      }
+
       dequeueSeek();
       isLive = isLive || _video.currentTime === Infinity; // Just in case backend and video metadata disagree about this
       loaded = true;
     }, this);
 
+    /**
+     * Fired when there's a change on audioTracks
+     * @private
+     * @method OoyalaVideoFactory#onAudioChange
+     * @callback OoyalaVideoFactory#raiseAudioChange
+     */
+    var _onAudioChange = _.bind(function(event) {
+      var audioTracks = this.getAvailableAudio();
+      raiseAudioChange(audioTracks);
+    }, this);
+    
+    /**
+     * Raised notification to VideoController
+     * @private
+     * @method OoyalaVideoFactory#onAudioChange
+     * @fires VideoController#EVENTS.MULTI_AUDIO_CHANGE
+     */
+    var raiseAudioChange = _.bind(function(audioTracks) {
+      // the problem here is that onchange gets triggered twice so
```

Comment:
What's triggering onchange twice? Just out of curiosity.

Reply:
I think it's triggered twice because we have two consecutive changes. 
https://github.com/crosswalk-project/chromium-crosswalk/blob/master/third_party/WebKit/LayoutTests/media/avtrack/audio-track-enabled.html